modules/rostests/apitests/crt/math_helpers.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Helpers for testing math functions
+ * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#pragma once
+
+#define _USE_MATH_DEFINES
+#include <math.h>
+#include <float.h>
+#include <apitest.h>
+
+static
+__inline
+double
+u64_to_dbl(UINT64 x)
+{
+    return *(double*)(&x);
+}
+
+static
+__inline
+UINT64
+dbl_to_u64(double x)
+{
+    return *(UINT64*)(&x);
+}
+
+static
+__inline
+float
+u32_to_flt(UINT32 x)
+{
+    return *(float*)(&x);
+}
+
+static
+__inline
+UINT32
+flt_to_u32(float x)
+{
+    return *(UINT32*)(&x);
+}
+
+typedef struct _TESTENTRY_DBL
+{
+    unsigned long long x;
+    unsigned long long result;
+} TESTENTRY_DBL;
+
+typedef struct _TESTENTRY_FLT
+{
+    unsigned long x;
+    unsigned long result;
+} TESTENTRY_FLT;
+
+#define ok_eq_dbl_exact_(file, line, func, ullx, z, ullexp) \
+    { \
```

Comment:
Not using the `do { .... } while (0)` macro trick to make the macro "behave" a bit more like a function?

Reply:
What does that change?